timesketch/tsctl.py

```diff
@@ -442,3 +443,40 @@ def export_sigma_rules(path):
             fw.write(rule.rule_yaml.encode("utf-8"))
         n = n + 1
     print(f"{n} Sigma rules exported")
+
+
+@cli.command(name="info")
+def info():
+    """Get various information about the environment that runs Timesketch."""
+
+    # Get Timesketch version
+    print(f"Timesketch version: {version.get_version()}")
+
+    # Get plaso version
+    output = subprocess.check_output(["psort.py", "--version"])
```

Comment:
hm, I assume if that is the case then your installation is broken anyway, as TS without Plaso does not make that much sense, so I think I would keep it simple and not check for this?

Reply:
Under the aspect of using the command mainly to support users with issues, it will work like this as well. And a breaking tsctl command does not ruin much and also tells a story to debug :)